### PR TITLE
Use graphql v14.0.0

### PR DESCRIPTION
**Pull request by kamilkisiela**
I removed `packages/*/package-lock.json` because it's not something lerna produce and it's just trash. Someone just ran `npm install` in subdirectories and that's why it got here.

`apollo-cache-inmemory` had no `^14.0.0` in peerDeps.

Related PR: apollographql/apollo-link#777
Apollo Angular works with new graphql too: apollographql/apollo-angular#795

@hwillson one question, have you guys changed renovate's configuration to not create PRs? I see there's a branch available.

### CODE REVIEW

File: packages/apollo-cache-inmemory/package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "graphql-anywhere": "file:../graphql-anywhere"
   },
   "peerDependencies": {
-    "graphql": "0.11.7 || ^0.12.0 || ^0.13.0"
+    "graphql": "0.11.7 || ^0.12.0 || ^0.13.0 || ^14.0.0"
```
Comment (jaydenseric) [inline packages/apollo-cache-inmemory/package.json]: `"0.11.7 - 14"` is more succinct and easier to update.

**Comment by raja** [comment]
What is that status of this PR ?

**Comment by daggerjames** [comment]
duplicated with #3978

**Comment by hwillson** [comment]
This was addressed in https://github.com/apollographql/apollo-client/commit/b082c8b23e791cfdca0fc0680a33ec542beb0a68. Thanks @kamilkisiela!

**Pull request closed without merging**